eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         BeforePublishSteps: ${{ parameters.BeforePublishSteps }}
 
   - job: 'Analyze'
+    condition: ne(variables['Skip.Analyze'], 'true')
```

Comment:
I can't recall off the top of my head how to do this properly but it might make sense to have these be template parameters

Reply:
We need to be able to inject them as variables at queue time.

Reply:
Conditions are evaluated before each stage/job/step starts - so can't be templated.